cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -55,6 +56,33 @@
     TENANT_DEPLOYMENTS_PATH + '/{name}'
 )
 
+MANAGER_VERSIONS_LIST = (
+    '3.4.2',
+    '4.0',
+    '4.0.1',
+    '4.1',
+    'master'
+)
+# Only test 3.4.2 and 4.0 with a single tenant on premium.
+# Everything else should work with multi tenant
+MANAGER_SINGLE_TENANT_SLICE = 2
```

Comment:
I agree with Lukasz. Having two lists feels more readable, especially since we have `get_single_tenant_versions_list` and `get_multi_tenant_versions_list` anyway